peachjam/migrations/0047_taxonomy_slug.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.16 on 2023-02-06 10:24
+
+from django.db import migrations
+
+from peachjam.models import Taxonomy
+
```

Comment:
@longhotsummer Not sure if its okay to import the model directly like this in the migration. The get_tree method doesn't work when I use `apps.get_model("peachjam", "Taxonomy")` I want the update to traverse the tree starting with the topmost level

Reply:
We just need a mechanism to apply this globally without having to do it manually.

We can add this (which you're right, isn't actually good practice) and do the migration, and then remove this migration.

The alternative is we do it entirely outside of the codebase and simply run a script in a shell after deploying. That's also reasonable, and probably easier?

Reply:
I'd rather run the script externally. We'd only have to do it for agp and lawlibrary right?. Do the other liis have any taxonomies yet?

Reply:
I think only those two. Sounds good.